### PR TITLE
XWIKI-19469: The size of the image download and open Lightbox links is too small

### PR DESCRIPTION
**Pull request by Sereza7**
**Jira:** https://jira.xwiki.org/browse/XWIKI-19469
**PR Changes:**
* Reduced margins and increased size of elements up to 44*44

See jira for details about the choice of solution.

**View:**
19469-WithoutMarginAndTabs
With tabbing order displayed by Firefox accessibility tool, we can see that the interactive elments cover most of the popup:
19469-WithoutMargin

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/less/lightbox.less [hunk 1]
```diff
@@ -81,8 +81,14 @@
   display: none;
 }
 .imageToolbar {
-  > * {
-    margin: .5em;
+  display: flex !important;
```
[the remark this answers is not in the document]
Comment (Sereza7) [reply xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/less/lightbox.less:84]: Thanks! I'll look for a workaround to not use it :+1:

[the remark this answers is not in the document]
Comment (Sereza7) [reply xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/less/lightbox.less:84]: I relooked at the situation and there's definitely multiple correct ways to do this properly :)

**Comment by Sereza7** [comment]
Created a forum post about the topic discussed for this PR. Keeping it on hold until there's some feedback. :)
https://forum.xwiki.org/t/css-recommendation-pointer-targets-are-large-enough/12241

File: xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/less/lightbox.less [hunk 2]
```diff
@@ -81,8 +81,14 @@
...
+
+  & > *, & a {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-width: 44px;
```
[the remark this answers is not in the document]
Comment (Sereza7) [reply xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/less/lightbox.less]: By the way, I didn't change the size of the icons themselves, but only the size of their targets.
See there for an illustrated example https://ishadeed.com/article/clickable-area/#hamburger-menus
In this case I could use larger icons but I think only increasing the target size is the most important part (icons are already the same size/larger than text content so should be just as legible).

**Comment by Sereza7** [comment]
**PR Changes:**
* Using skin defined values for size instead of hard coded ones.

*Note:* Only merge this commit after the values themselves are merged in https://github.com/xwiki/xwiki-platform/pull/2144

File: xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/less/lightbox.less [hunk 3]
```diff
@@ -80,19 +80,31 @@
...
+    & > *, & a {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      min-width: @target-size-recommended;
+      min-height: @target-size-recommended;
+    }
+
+    .tooltip-inner {
+      white-space:nowrap;
```
Comment (manuelleduc) [inline xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/less/lightbox.less:107]:
```suggestion
      white-space: nowrap;
```

**Pull request merged**